scripts/floater_convert

```diff
@@ -18,10 +18,10 @@ parser.set_defaults(progress=False)
 parser.add_argument('--input_dir', default='./', metavar='DIR',
                    help='directory in which to find the MITgcm float data')
 
-parser.add_argument('--output_format', choices=['pytables','bcolz','pandas'],
+parser.add_argument('--output_format', choices=['pytables','bcolz','pandas','netcdf'],
                     default='pytables', metavar='FMT',
                     help="output format of conversion "
-                         "('pytables', 'bcolz', or 'pandas')")
+                         "('pytables', 'bcolz', 'pandas', or 'netcdf')")
```

Comment:
It seems like we also need to add an `--input_format` option, which can be either `'binary'` or `'csv'`.

What would happen if you tried to use `--output_format='netcdf'` with `--input_format='binary'`. Would it break?